packages/app/src/components/DownloadMenu/DownloadMenu.js

```diff
@@ -35,40 +24,37 @@ import {
     apiDownloadTable,
 } from '../../api/analytics'
 import MenuButton from '../MenuButton/MenuButton'
+import MoreHorizontalIcon from '../../assets/MoreHorizontalIcon'
+import styles from './styles/DownloadMenu.module.css'
+
+const DenseMenuItem = ({ Icon, children, ...rest }) => (
```

Comment:
Separate to another file? Or is it ok to keep more than 1 component per file?

Reply:
This is not likely going to be used anywhere else in DV, and it's small, so I guess it's ok to keep it here.